conf/core/Testing.conf

```diff
@@ -1,22 +1,25 @@
 {
```

Comment:
Just alphabetized this in addition to the `GraphComparator` thread option addition.